src/Widgets/WindowClone.vala

```diff
@@ -41,6 +41,59 @@ namespace Gala {
         }
     }
 
+    /**
+     * Border to show around the selected window when using keyboard navigation.
+     */
+    class ActiveShape : Actor {
+        private Clutter.Canvas background_canvas;
+        private static int border_radius;
+
+        static construct {
+            var label_widget_path = new Gtk.WidgetPath ();
+            label_widget_path.append_type (typeof (Gtk.Label));
+
+            var tooltip_style_context = new Gtk.StyleContext ();
+            tooltip_style_context.add_class (Granite.STYLE_CLASS_CARD);
+            tooltip_style_context.add_class (Granite.STYLE_CLASS_ROUNDED);
+            tooltip_style_context.set_path (label_widget_path);
+
+            border_radius = tooltip_style_context.get_property (
+                Gtk.STYLE_PROPERTY_BORDER_RADIUS,
+                Gtk.StateFlags.NORMAL
+            ).get_int ();
```

Comment:
Since this border is spaced a bit outside the window, to make it look like a single width all the way through we need this border radius to be much larger. 
```suggestion
            ).get_int () * 4;
```

see the comparison:
![Screenshot from 2021-05-07 12 02 14](https://user-images.githubusercontent.com/7277719/117496653-33f44380-af2c-11eb-85ed-86a3485d7713.png)
![Screenshot from 2021-05-07 12 01 44](https://user-images.githubusercontent.com/7277719/117496655-35257080-af2c-11eb-9a28-315f3b2e967b.png)